pkg/pb/node.proto

```diff
@@ -80,3 +81,13 @@ message NodeRestrictions {
     int64 free_bandwidth = 1;
     int64 free_disk = 2;
 }
+
+// NodeVersion contains
+message NodeVersion {
+    int64 major = 1;
+    int64 minor = 2;
+    int64 patch = 3;
```

Comment:
i don't know if you saw the conversation with @brimstone in #engineering, but i think we need to make the version a string, so we can in the future support versions like "v3.0.0-beta1".

Reply:
done